### DISCUSSION TITLE
Mod Pack Inheritance

### DISCUSSION THREAD

**Discussion by ghost** [modrinth/.github#166] [Feature requests]
Mod packs should be able to inherit mods from other modpacks. This would allow you to make a mod pack that has some common mods like sodium and JEI and use them in other mod packs. By separating these mods it can make the mod pack seem less overwhelming. It can also be used as a way of grouping mods. So in large mod packs, you could see all of the terrain mods and all of the graphical mods etc.

**Comment by triphora** [comment]
Related to #94
